beangulp/importers/csvbase.py

```diff
@@ -121,6 +122,8 @@ def __init__(self, name, frmt='%Y-%m-%d'):
         self.frmt = frmt
 
     def parse(self, value):
+        if not value:
+            return None
```

Comment:
Why is returning `None` acceptable here? If the input is not a valid date an error should be raised.

Reply:
A CSV file can have multiple Date columns, e.g. posted date and transaction date. The posted date is used for `entry.date` and the transaction can be added to `entry.meta` in `finalize()`. But such a Date column might be empty for some transactions (e.g. `PAYMENT RECEIVED`). If an error is raised here, user is unable to process such rows in `finalize()`.

Reply:
Sure, that is a valid use case, but it should not be what the column adapter does by default. There must be a `default` parameter that can be passed to the constructor to enable returning a default when the value is the empty string. See how this is done for the generic `Column` class. That mechanism should be exposed here and extended to allow to use `None` as a default value. I'll implement this ASAP.

Reply:
Thanks for implementing the default value. It works great!